xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/config/HibernateConfig.java

```diff
@@ -0,0 +1,77 @@
+package com.icthh.xm.commons.domain.event.config;
+
+import com.icthh.xm.commons.domain.event.domain.Outbox;
+import com.icthh.xm.commons.domain.event.repository.OutboxRepository;
+import org.hibernate.MultiTenancyStrategy;
+import org.hibernate.cfg.Environment;
+import org.hibernate.context.spi.CurrentTenantIdentifierResolver;
+import org.hibernate.engine.jdbc.connections.spi.MultiTenantConnectionProvider;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.AutoConfigurationPackages;
+import org.springframework.boot.autoconfigure.domain.EntityScanPackages;
+import org.springframework.boot.autoconfigure.orm.jpa.JpaProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.util.StringUtils;
+
+import javax.sql.DataSource;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Configuration
+@EnableJpaRepositories(basePackageClasses = OutboxRepository.class)
+public class HibernateConfig {
+
+    @Autowired
+    private JpaProperties jpaProperties;
+
+    @Autowired
+    private BeanFactory beanFactory;
+
+    @Bean
+    JpaVendorAdapter jpaVendorAdapter() {
+        return new HibernateJpaVendorAdapter();
+    }
+
+    @Bean
+    @Primary
+    LocalContainerEntityManagerFactoryBean entityManagerFactory(DataSource dataSource,
```

Comment:
In commons v3.0 this config should be already in commons (not in any microservice)
Lets check and put TODO here to not forget to use one from commons v3.